test/lib/general.lib.test.js

```diff
@@ -48,4 +48,49 @@ describe('RequestLib', () => {
       expect(result).to.equal('2015-10-21T20:31:57.000Z')
     })
   })
+
+  describe('#periodsOverlap', () => {
+    let referencePeriod
+    let checkPeriod
+
+    describe('when given dates that do not overlap', () => {
```

Comment:
Because we name the params `fooPeriods` I'd suggest tweaking the `describe`

```suggestion
    describe('when given periods that do not overlap', () => {
```